dist/index.d.ts

```diff
@@ -9,23 +9,24 @@
 interface HTMLCanvasElement {
   getContext(
     contextId: "gpupresent"
-  ): GPUCanvasContext | null;
+  ):
+    | GPUPresentationContext
+    | GPUCanvasContext
+    | null;
 }
 
 interface OffscreenCanvas {
   getContext(
     contextId: "gpupresent"
-  ): GPUCanvasContext | null;
+  ):
+    | GPUPresentationContext
+    | GPUCanvasContext
+    | null;
 }
 
 type GPUExtent3DStrict =
   | Iterable<GPUIntegerCoordinate>
-  | GPUExtent3DDictStrict;
-interface GPUExtent3DDictStrict
-  extends GPUExtent3DDict {
-  /** @deprecated */
-  depth?: undefined;
```

Comment:
I'm not sure what good it does to keep this indefinitely? It's already been removed from the spec and Chrome, so indicating to users here that's it's available and merely deprecated seems actively harmful?

Also, if this is the only reason we have GPUExtent3DStrict then I'm in favor of removing it.

Reply:
It has type `undefined`, which will cause a type check failure if someone does make the mistake of typing `depth:`. Without this due to the type checking rules (since depthOrArrayLayer is optional), `{width: 2, height: 2, depth: 2}` wouldn't cause any errors but depth would be silently ignored.

Reply:
Ah, got it. My lack of Typescript knowledge is showing. :) That does sound desirable I'll put this back with a comment to explain the logic behind keeping it, then merge the change.

Reply:
Good idea, thanks for adding the comment